configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
 	shadow
 	missing-prototypes
 	missing-declarations
+	suggest-attribute=noreturn
+	suggest-attribute=format
 	strict-prototypes
-	declaration-after-statement
 	pointer-arith
 	write-strings
 	cast-align
 	bad-function-cast
 	missing-format-attribute
+	float-equal
 	format=2
-	format-security
-	no-format-nonliteral
-	no-long-long
-	unsigned-char
-	gnu89-inline
-	no-strict-aliasing
+	format-signedness
+	shift-overflow=2
+	overlength-strings
```

Comment:
Prevents defining raw string literals that exceed the length defined as the "minimum maximum" in the C standard. No strings in libqb exceed this, but to ensure no submissions that break future compatibility with platforms that have different support, this will catch that.